react/components/Popover.js

```diff
@@ -6,6 +6,7 @@ export default class Popover extends Component {
   static propTypes = {
     /** Function that will display the header */
     renderHeader: PropTypes.func.isRequired,
+    arrowUpColor: PropTypes.string
```

Comment:
`classes` or `additionalClass` instead. `arrowUpColor` makes the impression to accept values like `#ccc` or `rgba(20,05,08,1)`, and not a CSS class.